packages/loot-core/src/server/budget/goaltemplates.js

```diff
@@ -53,120 +68,56 @@ async function processTemplate(month, force) {
-    },
-    {
-      type: 'weeks',
-        re: /^#template \$?(\d+(\.\d{2})?) repeat every (\d+) weeks starting (\d{4}\-\d{2}\-\d{2})$/im,//eslint-disable-line
-      params: ['amount', null, 'weeks', 'starting']
-    },
-    {
-      type: 'weeks',
-        re: /^#template \$?(\d+(\.\d{2})?) repeat every (\d+) weeks starting (\d{4}\-\d{2}\-\d{2}) up to \$?(\d+(\.\d{2})?)$/im,//eslint-disable-line
-      params: ['amount', null, 'weeks', 'starting', 'limit']
-    },
-    {
-      type: 'by_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) repeat every year$/im,//eslint-disable-line
-      params: ['amount', null, 'month']
-    },
-    {
-      type: 'by_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) repeat every (\d+) years$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'repeat']
-    },
-    {
-      type: 'spend',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2})$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from']
-    },
-    {
-      type: 'spend',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2}) repeat every (\d+) months$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from', 'repeat']
-    },
-    {
-      type: 'spend_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2}) repeat every year$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from']
-    },
-    {
-      type: 'spend_annual',
-        re: /^#template \$?(\d+(\.\d{2})?) by (\d{4}\-\d{2}) spend from (\d{4}\-\d{2}) repeat every (\d+) years$/im,//eslint-disable-line
-      params: ['amount', null, 'month', 'from', 'repeat']
-    },
-    {
-      type: 'percentage',
-      re: /^#template (\d+(\.\d+)?)% of (.*)$/im,
-      params: ['percent', null, 'category']
-    },
-    { type: 'error', re: /^#template .*$/im, params: [] }
-  ];
+const TEMPLATE_PREFIX = '#template ';
 
+async function getCategoryTemplates() {
   let templates = {};
 
-  let notes = await db.all(`SELECT * FROM notes WHERE note like '%#template%'`);
+  let notes = await db.all(
+    `SELECT * FROM notes WHERE note like '%${TEMPLATE_PREFIX}%'`
+  );
 
   for (let n = 0; n < notes.length; n++) {
     let lines = notes[n].note.split('\n');
     let template_lines = [];
     for (let l = 0; l < lines.length; l++) {
-      for (let m = 0; m < matches.length; m++) {
-        let arr = matches[m].re.exec(lines[l]);
-        if (arr) {
-          let matched = {};
-          matched.line = arr[0];
-          matched.type = matches[m].type;
-          for (let p = 0; p < matches[m].params.length; p++) {
-            let param_name = matches[m].params[p];
-            if (param_name) {
-              matched[param_name] = arr[p + 1];
-            }
-          }
-          template_lines.push(matched);
-          break;
-        }
+      let line = lines[l].trim();
```

Comment:
This line would need to be update too if we do this.  

let line = lines[l].trim().toLowerCase();